indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -108,12 +129,93 @@
+                        contentType: 'application/json',
+                        error: handleAjaxError,
+                        complete: IndicoUI.Dialogs.Util.progress()
+                    });
+                }
+
+                $('.session-item-picker').itempicker({
+                    containerClasses: 'session-item-container',
+                    filterPlaceholder: $T.gettext('Search for a session'),
+                    createCustomFooterElements: function(container) {
+                        var self = this;
+                        var addNewSession = $('<div>', {
+                            'class': 'action-button',
+                            'text': $T.gettext('Add new session')
+                        });
+
+                        addNewSession.on('click', function() {
+                            self.itempicker('hide');
+                            ajaxDialog({
+                                title: $T.gettext('Add new session'),
+                                url: {{ url_for('sessions.create_session', event) | tojson }},
+                                onClose: function(data) {
+                                    if (data) {
+                                        $('.session-item-picker').itempicker('refreshItemList', data.sessions);
+                                    }
+                                }
+                            });
+                        });
+
+                        container.append(addNewSession);
+                    },
+                    createItem: function(itemData) {
+                        var item = $('<div>', {'class': 'dropdown-item', 'data': {'filter': itemData.title},
+                                               'title': itemData.title});
+                        item.append($('<span>', {'class': 'session-bgcolor-preview',
+                                                 'css': {'background': '#' + itemData.colors.background}}));
+                        item.append($('<span>', {'class': 'session-title', 'text': itemData.title}));
+                        return item;
+                    },
+                    onSelect: function(session) {
+                        var $this = $(this);
+                        var styleObject = $this[0].style;
+                        patchObject($this.data('href'), $this.data('method'), {session_id: session.id});
+                        $this.find('.label').text(session.title);
+                        styleObject.setProperty('color', '#' + session.colors.text, 'important');
+                        styleObject.setProperty('background', '#' + session.colors.background, 'important');
+                    },
+                    onClear: function(session) {
+                        var $this = $(this);
+                        patchObject($this.data('href'), $this.data('method'), {detach_session: true});
+                        $this.find('.label').text($T.gettext('Sessions'));
+                        $this.removeAttr('style');
```

Comment:
IMO it's better to toggle a class which holds all the styling, instead of adding/removing the style attribute. Hope you didn't get sick of me and classes yet :tongue: